### ISSUE TITLE
[A11Y] <amp-bird-player>'s buttons are not accessible.

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5497]
bird-players needs to use `role=button` and proper `title` attributes such as `Play video` on its UI controls (which are just `div`s) Currently even the `play` action is not accessible in TalkBack.

Not specific to the AMP component but a general a11y issues with bird-player. 

@DarXector Do you work for brid.tv or do you know of any contacts there? Thanks!

**Comment by adelinamart** [comment]
Is this issue still blocked?

**Comment by DarXector** [comment]
This issue somehow slipped by me. I will forward this to the Brid Development team and it will be in the next release.

**Comment by aghassemi** [comment]
@DarXector Thanks!
